### PR TITLE
Remove packer

### PR DESCRIPTION
**Pull request by liztio**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ REGION=us-west-2
 AZ=us-west-2b
 
 # What to name your CloudFormation stack
-STACK=Heptio-Kubernetes
+STACK=Heptio-Kubernete
```
Comment (chuckha) [inline README.md]: dropped an S

File: README.md [hunk 2]
```diff
@@ -85,7 +85,7 @@ aws cloudformation create-stack \
 
 To deploy your own changes manually from source, you'll need to upload the contents of the `scripts` and `templates` directories to S3, and configure your CloudFormation to use those S3 URL's.
 
-If you're making changes to things like the Kubernetes version or anything installed in the base AMI, you'll also need to rebuild the AMI with Packer.  See the "Local development" section below for more details.
+If you're making changes to things like the Kubernetes version or anything installed in the base AMI, you'll also need to rebuild the AMI with Wardroom.  See the "Local development" section below for more details.
```
Comment (chuckha) [inline README.md]: Add link to wardroom: https://github.com/heptiolabs/wardroom

File: README.md [hunk 3]
```diff
@@ -152,26 +152,37 @@ kubectl get nodes
...
+  from a base AMI, which you can recreate using
+  [wardroom](https://github.com/heptiolabs/wardroom).
+- `kubernetes-cluster-with-new-vpc.template` sets up a new VPC with a public and
+  private subnets, and calls out to `kubernetes-cluster.template` as a
+  sub-stack.
 
 **The `scripts` directory**:
 
-This contains files that required by the templates.  This is to avoid inlining a lot of text directly in the template files.  Templates access files from this directory by referencing their S3 URL's, which is why this directory and the `templates` directory both need to be copied into S3 in order to be used.
+This contains files that required by the templates. This is to avoid inlining a
```
Comment (chuckha) [inline README.md]: should be

> that are required

(although I recognize you didn't write this, so if it makes sense to open a separate ticket for proof reading existing documentation that's fine by me)

File: README.md [hunk 4]
```diff
@@ -152,26 +152,37 @@ kubectl get nodes
...
-
-- `create-ami.sh`: This file launches packer, configured to launch an instance in `us-east-2` based on a stock Ubuntu 16.04 LTS image, installing Kubernetes, Docker, and various other components needed for the cluster to run.  Packer will output the resulting AMI when complete.
-- `payload/prepare-ami.sh`: This file is the script run by packer on the EC2 instance, and is responsible for actually installing Kubernetes and the various other components.  This is where we update the version of kubernetes baked into the AMI.
-- `deploy-ami.sh`: This is responsible for taking an AMI that was created by Packer and copying it into each region.  It also outputs YAML suitable for copying into `templates/kubernetes-cluster.template` to configure the cluster with the right AMI ID for each region.
+- `setup-k8s-master.sh.in`: This file is run by the master node on first boot as
+  part of its initialization. It is a [Mustache
+  template](https://mustache.github.io/) file, with template variables that are
+  set by the `kubernetes-cluster.template` template. Note that this template
+  file does not install the Kubernetes binaries, those are baked into the AMI as
```
Comment (chuckha) [inline README.md]: could stop the sentence at AMI.

File: templates/kubernetes-cluster.template [hunk 5]
```diff
@@ -307,34 +307,36 @@ Mappings:
...
-      '64': ami-6369fa19
-    us-east-2:
-      '64': ami-e89eb08d
-    us-west-2:
-      '64': ami-7ed30d06
+      us-east-1:
+        '64': ami-957920ef
+      ap-south-1:
+        '64': ami-ea306785
+      eu-west-3:
```
Comment (chuckha) [inline templates/kubernetes-cluster.template]: ffff

You will have to leave this one out for now until this gets closed or our CI changes: https://github.com/boto/boto/pull/3784

Comment (liztio) [reply templates/kubernetes-cluster.template]: Will be solved once #126 is merged

**Pull request merged**